internal/fnruntime/runner.go

```diff
@@ -179,32 +179,35 @@ type FunctionRunner struct {
 
 func (fr *FunctionRunner) Filter(input []*yaml.RNode) (output []*yaml.RNode, err error) {
 	pr := printer.FromContextOrDie(fr.ctx)
+	var fnOutput string
+	defer func() {
+		pr.Printf(fnOutput)
```

Comment:
Do we need to use a defer for this? I usually think of it more for error handling and cleanup.

Reply:
Changed it to not use a defer